src/vue/common/Drawer.vue

```diff
@@ -150,7 +150,9 @@ $media-small: 460px;
 }
 
 .drawer-transition-leave-active {
-  animation-duration: .175s;
+  /* time animation-duration > animation,
+  because after closing it appears again*/
+  animation-duration: .100s;
```

Comment:
hmmm. should not it be bigger than animations of `.drawer__backdrop` and `.drawer__pane`? comments say this thing.
also do not use trailing zeros, `0.1` is much better

Reply:
No, ok